scss/styles.scss

```diff
@@ -0,0 +1,772 @@
+  .story {
+    border: 1px solid $body-bg;
+    margin: 1rem 0;
+  }
+
+  .top-story-h2 {
+    @include h2-style();
+  }
+
+  .caption {
+    @include h3-style();
+  }
+
+  h4 {
+    @include h4-style();
+
+    padding: 0 2rem;
+  }
+
+  .paragraph {
+    @include paragraph();
+
+    padding: 0 2rem;
+  }
+
+  .form {
+    background-color: #f4f5f6;
+    padding: 2rem;
+
+    .form-icon {
+      flex: none;
+      flex-grow: 0;
+      flex-shrink: 0;
+      flex-basis: auto;
+      display: block;
+      width: 29px;
+      height: 28px;
+      background: url("https://g.newsweek.com/www/images/logo-n1.svg")
+        center/contain no-repeat #f72210;
+      // background-image: url("https://g.newsweek.com/www/images/logo-n1.svg");
+      background-position-x: center;
+      background-position-y: center;
+      background-size: contain;
+      background-repeat: no-repeat;
+      background-attachment: initial;
+      background-origin: initial;
+      background-clip: initial;
+      background-color: rgb(247, 34, 16);
+      margin-right: 10px;
+    }
+
+    form {
+      input,
+      button {
+        padding: 1rem;
+      }
+
+      .btn-primary {
+        font-size: 2rem;
+        background-color: $bg-primary;
+        border: 0;
+        font-weight: bold;
+      }
+    }
+}
+
+.magazine {
+  @include section-spacing();
+
+  .magazine-article {
+    h4 {
+      @include h4-style();
+      font-size: 1.6rem;
+
+		h4:hover {
+			text-decoration: underline $primary;
+			cursor: pointer;
+		}
+
+		h3 {
+			@include h3-style();
+		}
+
+  }
+
+    h3 {
+      @include h3-style();
+    }
+
+    .feature {
+      font-size: 2rem;
+      border: 0;
+      font-weight: bold;
+    }
+
+    .caption {
+      width: max-content;
+      padding: 1rem 1rem 1rem 0;
+      font-weight: bold;
+      color: $bg-primary;
+
+      span {
+        background-color: $bg-primary;
+        color: #fff;
+        padding: 0.5rem;
+      }
+    }
+  }
+}
+
+.opinion {
+  // @include section-spacing();
+
+  .content {
+    grid-template-columns: auto auto;
+
+	h4:hover {
+		text-decoration: underline $primary;
+		cursor: pointer;
+	}
+
+	.paragraph {
+		@include paragraph();
+	}
+}
+
+.sports {
+	@include section-spacing();
+
+	h2 {
+		@include h2-style();
+	}
+
+	h3 {
+		@include h3-style();
+	}
+
+	h4 {
+		@include h4-style();
+	}
+
+	h4:hover,
+	h5:hover {
+		text-decoration: underline $primary;
+		cursor: pointer;
+	}
+
+	h5 {
+		@include h5-style();
+	}
+
+	.paragraph {
+		@include paragraph();
+	}
+}
+
+.contact {
+  padding: 4rem;
+
+  .subscribe {
+    h2 {
+      color: $primary;
+      font-weight: bolder;
+      font-size: 1.8rem;
+    }
+
+    .subscribe-content {
+      background-color: #f4f5f6;
+      padding: 4rem;
+
+      .subscribe-text {
+        h4 {
+          font-weight: bolder;
+          color: $body-color;
+          font-size: 3rem;
+        }
+
+        .btn-subscribe {
+          background-color: $bg-primary !important;
```

Comment:
Remove important

Reply:
Thanks for your reviews. They were really helpful.